posthog/models/team/team.py

```diff
@@ -123,6 +123,8 @@ class Team(UUIDClassicModel):
         "posthog.Dashboard", on_delete=models.SET_NULL, null=True, related_name="primary_dashboard_teams"
     )  # Dashboard shown on project homepage
 
+    actor_on_events_querying_setting_enabled: models.BooleanField = models.BooleanField(null=True, blank=True)
```

Comment:
Should this be set to true for new teams?

Reply:
Makes sense to me!